nbs/explains/background_tasks.ipynb

```diff
@@ -0,0 +1,296 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Background Tasks\n",
+    "\n",
+    "> Background tasks are functions run after handlers return a response."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Useful for operations where the user gets a response quickly but doesn't need to wait for the operation to finish. Typical scenarios include:\n",
+    "\n",
+    "- User setup in complex systems where you can inform the user and other people later in email that their account is complete\n",
+    "- Batch processes that can take a significant amount of time (bulk email or API calls)\n",
+    "- Any other process where the user can be notified later by email, websocket, webhook, or pop-up"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "::: {.callout-note}\n",
+    "Background tasks in FastHTML are built on Starlette's background tasks, but with some added sugar. Starlette's background task design is an easy-to-use wrapper around Python's async and threading libraries. Background tasks often make apps feel faster to the end user.\n",
```

Comment:
They don't just *feel* faster -- they *are* faster! The user experiences the app returning immediately, rather than waiting for the bg task to complete.